File: usdt.php
<?php

// URL API TokenView
$api_url = "https://api.tokenview.io/v1/transaction/USDT";  // Pastikan API URL yang benar

// Alamat wallet atau ID transaksi yang akan dicari
$address = "TRGSNhnQ9d6PwqB5Kd9NsRvXGYgbNRfnHP";  // Ganti dengan alamat wallet yang relevan

// Siapkan parameter API
$params = array(
    "address" => $address,
    "api_key" => "YOUR_API_KEY"  // Ganti dengan API key jika diperlukan
);

// Siapkan URL untuk permintaan API
$url = $api_url . "?" . http_build_query($params);

// Inisialisasi cURL
$ch = curl_init();

// Set opsi cURL untuk permintaan GET
curl_setopt($ch, CURLOPT_URL, $url);
curl_setopt($ch, CURLOPT_RETURNTRANSFER, 1);

// Eksekusi permintaan cURL
$response = curl_exec($ch);

// Cek jika ada kesalahan cURL
if (curl_errno($ch)) {
    echo "Error: " . curl_error($ch);
} else {
    // Tampilkan hasil respons API
    $data = json_decode($response, true);
    echo "Transaction Data: <pre>";
    print_r($data);
    echo "</pre>";
}

// Tutup koneksi cURL
curl_close($ch);

?>

File: usdt.py
import requests

# URL API TokenView (contoh, pastikan kamu memiliki API key atau dokumentasi API yang sesuai)
api_url = "https://api.tokenview.io/v1/transaction/USDT"  # Ini hanya contoh, pastikan URL API yang benar

# Ganti dengan alamat wallet atau ID transaksi
address = "TRGSNhnQ9d6PwqB5Kd9NsRvXGYgbNRfnHP"  # Contoh alamat wallet

# Parameter untuk permintaan API (misalnya, mencari data transaksi atau saldo)
params = {
    "address": address,
    "api_key": "YOUR_API_KEY"  # Jika perlu API key
}

# Kirimkan permintaan GET ke API
response = requests.get(api_url, params=params)

# Periksa apakah permintaan berhasil
if response.status_code == 200:
    # Menampilkan data hasil respons
    data = response.json()
    print("Transaction Data:", data)
else:
    print(f"Error: {response.status_code}")
